src/CompilationResult.php

```diff
@@ -71,101 +65,19 @@ public function getCss()
         return $this->css . $this->getSourceMapCss();
     }
 
-    /**
-     * Adds to list of parsed files
-     *
-     * @api
-     *
-     * @param string|null $path
-     *
-     * @return void
-     */
-    public function addParsedFile($path)
-    {
-        if (! \is_null($path) && is_file($path)) {
-            $this->parsedFiles[realpath($path)] = filemtime($path);
-        }
-    }
-
-    /**
-     * Returns list of parsed files
-     *
-     * @api
-     *
-     * @return array<string, int>
-     */
-    public function getParsedFiles()
-    {
-        return $this->parsedFiles;
-    }
-
-    /**
-     * Save the imported files with their resolving path context
-     * @param string $currentDirectory
-     * @param string $path
-     * @param string $filePath
-     *
-     * @return void
-     */
-    public function addImportedFile($currentDirectory, $path, $filePath)
-    {
-        $this->importedFiles[] = ['currentDir' => $currentDirectory, 'path' => $path, 'filePath' => $filePath];
-    }
-
-
-    /**
-     * Get the list of the already imported Files
-     * used by the compiler to check the once condition on @import
-     * @return string[]
-     */
-    public function getImportedFiles()
-    {
-        return array_column($this->importedFiles, 'filePath');
-    }
-
-    /**
-     * @return array
-     * @phpstan-return list<array{currentDir: string, path: string, filePath: string}>
-     *
-     * @internal
-     */
-    public function getImports()
-    {
-        return $this->importedFiles;
-    }
-
     /**
      * @return string[]
      */
     public function getIncludedFiles()
     {
-        return array_column($this->importedFiles, 'filePath');
+        return $this->getIncludedFiles();
```

Comment:
Mistake here I think. Should be `$this->includedFiles;`

Reply:
indeed